theme/templates/resource-landing-page/coverage.html

```diff
@@ -24,6 +24,8 @@ <h4>Spatial</h4>
                         {% if spatial_coverage.name %}
                             <div>
                                 <small>Place/Area Name: </small>
+                            </div>
+                            <div class="space-bottom">
```

Comment:
Well done. I noticed the space-bottom in the hydroshare-core.css.
.space-bottom {
	margin-bottom: 15px;
}